doc/source/ray-air/doc_code/hf_trainer.py

```diff
@@ -12,6 +12,10 @@
 from ray.train.huggingface import HuggingFaceTrainer
 from ray.air.config import ScalingConfig
 
+
+# If using GPUs, set this to True.
+use_gpu = False
```

Comment:
dumb, yet effective trick for cases you want to show users you can use gpus, but want cpus on CI:

```
use_gpu = True  # include in docs

use_gpu = False # exclude

<code using use_gpu>
```